data-structures-csharp/data-structures-csharp/RedBlackTree/NodeType.cs

```diff
@@ -8,7 +8,7 @@ namespace DataStructures.RedBlackTreeSpace
 {
     public partial class RedBlackTree<TKey, TValue>
     {
-        private enum NodeType
+        public enum NodeType
```

Comment:
Is it essential to expose `NodeType`? It should be internal detail of the implementation.